### DISCUSSION TITLE
Display affected row count in the query info table

### DISCUSSION THREAD

**Discussion by squatto** [spatie/laravel-ray#79] [Ideas]
When using `showQueries()`, the table below the query currently displays the connection name and the query execution time. It would be useful to also see the number of rows affected by the query.

Thanks!

**Comment by squatto** [comment]
Here is an example for context. I'd love to know how many rows were affected by the `update` query:

![image](https://user-images.githubusercontent.com/748444/104385751-09036100-54f1-11eb-9ba7-9a787c809ed6.png)

**Reply by freekmurze** [reply]
Good idea, I would accept a PR that already adds the number of affected rows to the `ExecutedQuery` payload.

**Reply by squatto** [reply]
I've looked around for more information on retrieving the number of affected rows, and it's unfortunately not as easy as I had hoped. Since you're relying on the `QueryExecuted` event to get the query and info, we're limited to only what's available in the event payload. That unfortunately doesn't include the affected rows. I dug into the Laravel database code and the way it handles running queries seemingly makes it difficult to capture the affected rows in a way that could make them available to the event. There's a reference to it [here](https://github.com/laravel/ideas/issues/1297). You can also check out the `affectingStatement()` and `run()` methods on the `\Illuminate\Database\Connection` class to see how the query is run.

For now I think that we can shelve this idea, since it's going to require a relatively non-trivial change to the framework itself.

**Reply by freekmurze** [reply]
Ok, thanks for the time you spent on investigating this.

**Reply by squatto** [reply]
My pleasure!
